gemini/sample-apps/fashion-trends-prediction/README.md

```diff
@@ -0,0 +1,200 @@
+   gcloud artifacts repositories create "$AR_REPO" --location="$LOCATION" --repository-format=Docker
+   gcloud auth configure-docker "$LOCATION-docker.pkg.dev"
+   gcloud builds submit --tag "$LOCATION-docker.pkg.dev/$PROJECT_ID/$AR_REPO/$SERVICE_NAME"
+   ```
+
+4. The final step is to deploy the service in Cloud Run with the image that we had built and had pushed to the Artifact Registry in the previous step:
+
+   In Cloud Shell, execute the following command:
+
+   ```bash
+   gcloud run deploy "$SERVICE_NAME" \
+     --port=8501 \
+     --image="$LOCATION-docker.pkg.dev/$PROJECT_ID/$AR_REPO/$SERVICE_NAME" \
+     --allow-unauthenticated \
+     --region=$LOCATION \
+     --platform=managed  \
+     --project=$PROJECT_ID \
+     --set-env-vars=GCP_PROJECT=$PROJECT_ID,GCP_REGION=$LOCATION
+   ```
+
+On successful deployment, you will be provided a URL to the Cloud Run service. You can visit that in the browser to view the Cloud Run application that you just deployed. Choose the functionality that you would like to check out and the application will prompt the Vertex AI Gemini API and display the responses.
+
+### How to create and run on your own data
+
+The current data is created by scraping posts from Instagram influencers and celebrities. It is stored in Google Cloud Storage and the exact path is specified in the app/config.py file as `config['Data']['current_data']`. The code uses the file stored in this location to generate trend predictions.
+
+If you want to create your own data (a different source from Instagram or a different set of accounts and posts), then replace the file at the location specified in `config['Data']['current_data']` or upload your file on the UI itself.
+
+The file app/data_prep/create_new_data.py can be used to generate the new data file in the required format. Add your Instagram account username and password in the app/config.py file before starting to scrape Instagram.
+
+Required JSON format -
+
+```json
+{
+
+    "finaldata": {
+        "country1": {
+            "category1": ["outfit1", "outfit2", "outfit3", ...],
+            "category2": [],
+            ...
+        },
+        "country2": {},
+        ...
+    },
+
+    "top_categories": {
+        "country1": ["category1", "category2", ...],
+        "country2": [],
+        ...
+    },
+
+    "articles": [["article_link", "article_summary", {
+        "category1": ["outfit1", "outfit2", "outfit3", ...],
+        "category2": [],
+        ...
+        }],
+        ...
+    ]
+}
+```
+
+To extend this, one can use Cloud Scheduler to do the scraping periodically (since we want the data to not be stale for this usecase) and store the results in a Cloud Storage bucket from where they will be read to predict upcoming trends.
+
+## Useful links
+
+[Slide deck](https://docs.google.com/presentation/d/1io-ks1vWeymwykUJE789_A-16apZm9GoatAtBClsB94/edit?usp=sharing&resourcekey=0-TqnMc8_S7nTXBHK-CtmoFw "Slide deck")
```

Comment:
Visible to anyone in alphabet

Reply:
Ok, since this is a public repository, the links will need to be visible to anyone globally. Can you publish them in a different place or with a non-google.com account?

Reply:
Not sure if we have permission to do that so removed the deck and video links from the README.